automation/check-patch.e2e-multus-functests.sh

```diff
@@ -66,8 +66,8 @@ EOF
 
 function __remove-test-manifests() {
     echo "Remove test manifests"
-    $KUBECTL delete -f ${TEST_YAML}
-    $KUBECTL delete -f ${CNI_PLUGIN_YAML}
+    $KUBECTL delete --ignore-not-found=true -f ${TEST_YAML}
+    $KUBECTL delete --ignore-not-found=true -f ${CNI_PLUGIN_YAML}
```

Comment:
The multus tests cleanup after themselves, so this is required - or, the function should not be called.

Reply:
You mean that is NOT requiered right ? calling the cleanup function

Reply:
yes. The script you're executing is a full blown test - it clean up after itself (i.e. removes the pods + net-attach-def).

You remove the call to the cleanup function, along with its definition.